glide.yaml

```diff
@@ -21,7 +21,7 @@ import:
 - package: github.com/go-ini/ini
   version: ^1.21.0
 - package: github.com/projectcalico/libcalico-go
-  version: 1.2.0
+  version: 9c7017a9f079eb344a01cc21b72cb4c36ab2270f
```

Comment:
I'm not sure what we've done historically.  Do we normally release libcalico-go RCs?  If so I suspect that would be the preferred approach.  But happy to go with whatever we've done historically.